kore/src/Kore/AST/Sentence.hs

```diff
@@ -269,6 +301,34 @@ instance Unparse (SentenceSymbol level patternType) where
             , unparse sentenceSymbolAttributes
             ]
 
+    unparse2
+        SentenceSymbol
+            { sentenceSymbolSymbol
+            , sentenceSymbolSorts
+            , sentenceSymbolResultSort
+            }
+      =     Pretty.fillSep
+                [ "symbol"
+                , unparse2 sentenceSymbolSymbol
+                , "\naxiom"
```

Comment:
Embedding a newline in the string this way won't preserve indentation when this is printed. This should be,
```.hs
Pretty.vsep
    [ Pretty.fillSep [ "symbol", unparse2 sentenceSymbolSymbol ]
    , Pretty.fillSep [ "axiom", ... ]
    ]
```